### PR TITLE
Create file1

### PR DESCRIPTION
**Pull request by aamitaz**
check and verify

### CODE REVIEW

File: file1 [hunk 1]
```diff
@@ -0,0 +1 @@
+file1
```
Comment (aamit12) [inline file1]: give the proper content of file

**Review by aamit12 — COMMENTED** [review]
give the proper output

File: file1 (hunk 1 above)
Comment (aamit12) [reply file1:1]: now i change the file please review and approve

Comment (aamitaz) [reply file1]: now put the content in file please verify

**Review by aamit12 — APPROVED** [review]
good work

**Pull request merged**